### PR TITLE
Add file backend test into CI

### PR DESCRIPTION
**Pull request by DavidPineauScality**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,6 @@
     "node-mocks-http": "^1.5.2"
   },
   "scripts": {
-    "circle_prelude": "mkdir -p $CIRCLE_TEST_REPORTS/unit",
```
Comment (MichaelZapataScality) [inline package.json]: Good catch. :shaved_ice:

**Comment by alexandre-merle** [comment]
LGTM :+1:

**Comment by MichaelZapataScality** [comment]
:+1:

**Comment by DavidPineauScality** [comment]
Thanks guys. Merging on behalf of @LaurenSpiegel  (as per our agreement of the previous night), without e2e since this change has no impact on end-to-end (file backend only, no code switch modified)

**Pull request merged**